datacommons/__init__.py

```diff
@@ -12,6 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+################################## IMPORTANT #################################
```

Comment:
can you add a to do to enforce this with cloud build? A simple bash script would work.

Reply:
Filed https://github.com/datacommonsorg/api-python/issues/149, referenced here as well.

Reply:
Just tested, it works fine. Both are able to do the same thing in the same runtime.